web/admin/src/Question.js

```diff
@@ -55,6 +55,13 @@ export default class Question extends PureComponent {
     ) {
       this.startTimer(nextProps)
     }
+    if (nextProps.startCountdown) {
+      this.startTimer(nextProps)
+    }
+    if (nextProps.startCountdown === false) {
```

Comment:
is `startDown` expected to be something other than `true` or `false`?  If not why not
```javascript
if (nextProps.startCountdown) {
  ...
} else {
  ...
}
```